eng/pipelines/official.yml

```diff
@@ -116,3 +116,25 @@ extends:
           workloadDropNames: ${{ parameters.workloadDropNames }}
           primaryVsInsertionBranches: ${{ parameters.primaryVsInsertionBranches }}
           secondaryVsInsertionBranches: ${{ parameters.secondaryVsInsertionBranches }}
+    - stage: Publish
+      displayName: Publish
+      dependsOn: Build
+      condition: succeeded()
+      jobs:
+      - deployment: Publish
+        environment: 'DotNet-SDK-Workloads'
```

Comment:
nit: single quotes are not necessary here as the value resolves as a string automatically.